drizzlepac/alignimages.py

```diff
@@ -900,7 +903,8 @@ def determine_fit_quality(imglist, filtered_table, catalogs_remaining, print_fit
         consistency_check = True
         rms_limit = max(item.meta['fit_info']['TOTAL_RMS'], 10.)
         if not math.sqrt(np.std(np.asarray(xshifts)) ** 2 + np.std(
-                         np.asarray(yshifts)) ** 2) <= (rms_limit / 1000.0) / (item.wcs.pscale):
+                         np.asarray(yshifts)) ** 2) <= (rms_limit / 1000.0) / (item.wcs.pscale) or \
+                         rms_ratio > 0.1:
             consistency_check = False
```

Comment:
It would be good to declare what appear to be "magic numbers" (10, 1000, 0.1) to make the code understandable.

Reply:
Fix included in 2d3949e .